Sources/FINNSetup/BackendModel/FilterSetup.swift

```diff
@@ -97,6 +97,10 @@ public struct FilterSetup: Decodable {
                     return nil
                 }
             } else {
+                guard data.queries.count > 0 else {
```

Comment:
I would probably use `isEmpty` here instead of `count`.
I'm not sure if the comment below is necessary in this situation, but it doesn't hurt to leave it 😄

Reply:
I agree 👍  

Removed the comment too, it should be understandable from just the code